vaadin-ordered-layout-flow-demo/pom.xml

```diff
@@ -31,6 +31,16 @@
             <artifactId>vaadin-ordered-layout-flow</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-checkbox-flow</artifactId>
+            <version>${project.version}</version>
```

Comment:
Done

Reply:
It may be better to use SNAPSHOTS instead. Like is done here: https://github.com/vaadin/vaadin-form-layout-flow/blob/master/vaadin-form-layout-flow-demo/pom.xml#L44-L48

Reply:
Done